3.9/programs-arangod-log.md

```diff
@@ -141,27 +141,34 @@ Format                  | Example                  | Description
 
 ## Escaping
 
-`--log.escape value`
-
-This option toggles the escaping of log output. 
-
-If set to `true`, the following characters in the log output are escaped:
-
-- the carriage return character (hex `0d`)
-- the newline character (hex `0a`)
-- the tabstop character (hex `09`)
-- any other characters with an ordinal value less than hex `20`
-
-If the option is set to `false`, no characters are escaped. Characters with
-an ordinal value less than hex `20` will not be printed in this mode but will
-be replaced with a space character (hex `20`).
+There are two flags for retaining or escaping control and unicode 
+characters in the log. The flag `--log.escape` is deprecated since v3.9.0, and instead, 
+the new flags `--log.escape-control-chars` and `log.escape-unicode-chars` 
+should be used. 
+
+- `--log.escape-control-chars`: this flag applies to the control characters, which have hex codes below `\x20`, 
+  and also the character DEL with hex code `\x7f`. When the flag value is set to `false`, control 
+characters will be retained when they have a visible representation, and replaced
+with a space character in case they don't have a visible representation. For example, 
+the control characer `\n` is visible, so a `\n` will be displayed in the log. Contrary, 
+the control character `BEL` is not visible, so a space will be displayed instead.
+When the flag value is set to `true`, the hex code for the character is displayed, for 
+example, the `BEL` character will be displayed as its hex code, `\x07`.
+The default value for this flag is `true` to ensure compatibility with 
+previous versions.
```

Comment:
```suggestion
  characters will be retained when they have a visible representation, and replaced
  with a space character in case they don't have a visible representation. For example, 
  the control characer `\n` is visible, so a `\n` will be displayed in the log. Contrary, 
  the control character `BEL` is not visible, so a space will be displayed instead.
  When the flag value is set to `true`, the hex code for the character is displayed, for 
  example, the `BEL` character will be displayed as its hex code, `\x07`.
  The default value for this flag is `true` to ensure compatibility with 
  previous versions.
```